config/locales/en/settings.yml

```
@@ -91,6 +91,8 @@ en:
       google_login_description: "Allow users to sign up with their Google Account"
       wordpress_login: "Wordpress login"
       wordpress_login_description: "Allow users to sign up with their Wordpress Account"
+      oidc_login: "OIDC login"
+      oidc_login_description: "Allow users to sign up with OIDC"
```

Since administrators might not know what OIDC stands for, maybe we could include the full name:

```suggestion
      oidc_login: " OpenID Connect login"
      oidc_login_description: "Allow users to sign up with OpenID Connect (OIDC)"
```

I'm wondering what to do about the button to sign in with OIDC :thinking:. I guess we can leave "OIDC" for now and change it later if necessary.